src/app/projects/add-edit-project/add-edit-project.component.ts

```diff
@@ -292,6 +293,16 @@ export class AddEditProjectComponent implements OnInit, AfterViewInit, OnDestroy
     return this.myForm.get('logos') as FormArray;
   }
 
+  /**
+   * Getter to be able to access the contactFormEmails FormControl
+   * as a FormArray.
+   *
+   * @returns {void}
```

Comment:
the comment states it returns {void}, but it returns FormArray. Recommended to change for docs accuracy

Reply:
Thanks for this catch, I missed this!